pyroclast/run.py

```diff
@@ -1,38 +1,41 @@
-import multiprocessing
+import json
+import os
 import sys
 from importlib import import_module
 
 import tensorflow as tf
+from tensorflow.python.client import device_lib
 
-import pyroclast
-from pyroclast.common.tf_util import get_session
-from pyroclast.common.data_registry import get_dataset, split_dataset, dataset_iterator
 from pyroclast.common.cmd_util import common_arg_parser, parse_unknown_args
+from pyroclast.common.tf_util import setup_tfds
+from pyroclast.common.datasets import check_datasets, get_dataset_builder
 
+tf.compat.v1.enable_eager_execution()
```

Comment:
Some weird compatabilty thing I take it? For baselines, or what?

Reply:
To make sure that, even if someone has TF1 installed, that everything will run as well as it can. There's no reason not to have it (like using six for python code)